ApiReview/API.Polly.RateLimiting/NoDocs/Polly.RateLimiting/RateLimiterStrategyOptions.cs

```diff
@@ -0,0 +1,19 @@
+// Assembly 'Polly.RateLimiting'
+
+using System;
+using System.ComponentModel.DataAnnotations;
+using System.Runtime.CompilerServices;
+using System.Threading.RateLimiting;
+using System.Threading.Tasks;
+using Polly.Strategy;
+
+namespace Polly.RateLimiting;
+
+public class RateLimiterStrategyOptions : ResilienceStrategyOptions
+{
+    public sealed override string StrategyType { get; }
+    public Func<OnRateLimiterRejectedArguments, ValueTask>? OnRejected { get; set; }
+    [Required]
+    public RateLimiter? RateLimiter { get; set; }
```

Comment:
Why is RateLimitor [Required] when it's also nullable?

Reply:
It means "there is no default, and it must be not-null". In other words, this forces the customer to specify this in all cases.

Reply:
Makes perfect sense, thank you.